src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -421,6 +461,7 @@ export { ActiveFocusInvestigation };
 
 /** interface to describe props from mapStateToProps */
 interface DispatchedStateProps {
+  jurisdictionParentId: string;
```

Comment:
What caused the need for this?

Reply:
I need to use it in the component when making the call to `makePlansArraySelector()`

Reply:
What value of `jurisdictionParentId` are you going to pass to it?  Also, are we 100% certain that `makePlansArraySelector()` requires `jurisdictionParentId`?

Reply:
> What value of `jurisdictionParentId` are you going to pass to it? Also, are we 100% certain that `makePlansArraySelector()` requires `jurisdictionParentId`?

Was trying to achieve what this function is doing but with an added filter of `title`

```
getPlansArray(
    state,
    InterventionType.FI,
    [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
    CASE_TRIGGERED,
    [],
    jurisdictionParentId
  )
```